internal/services/azure_offering_template_service.go

```diff
@@ -0,0 +1,49 @@
+package services
+
+import (
+	"net/http"
+	"text/template"
+
+	"github.com/RHEnVision/provisioning-backend/internal/config"
+	"github.com/RHEnVision/provisioning-backend/internal/payloads"
+)
+
+var LighthouseTemplateFilePath = "./docs/lighthouse_template.json"
```

Comment:
Here you can just embed this. [Example](https://github.com/hardcaporg/hardcap/blob/main/internal/srv/kickstart.go#L45) from my prototype.

Reply:
I would like to keep it in the docs, I don't have a strong preference for it, but my gut feeling was docs should be first class citizen. But if we have strong preference for embedding, I'm not against, here it make sense IMHO, just the placing of it is bit of a blocker in my mind, but not necessarily ofc :)
Link in the docs still does the trick.

Reply:
Can we please use embedding? If you really want this to be physically present also in the docs folder, then make a relative symlink.